stdlib/src/collections/inline_list.mojo

```diff
@@ -98,3 +156,42 @@ struct InlineList[ElementType: CollectionElement, capacity: Int = 16](Sized):
         """Destroy all the elements in the list and free the memory."""
         for i in range(self._size):
             destroy_pointee(UnsafePointer(self._array[i]))
+
+    fn __iter__(
+        self: Reference[Self, _, _],
+    ) -> _InlineListIter[ElementType, capacity, self.is_mutable, self.lifetime]:
+        """Iterate over elements of the list, returning immutable references.
+
+        Returns:
+            An iterator of immutable references to the list elements.
+        """
+        return _InlineListIter(0, self)
+
+    @always_inline
+    fn __contains__[
+        C: ComparableCollectionElement
+    ](self: Self, value: C) -> Bool:
+        """Verify if a given value is present in the list.
+
+        ```mojo
+        var x = InlineList[Int](1,2,3)
+        if 3 in x: print("x contains 3")
+        ```
+        Parameters:
+            C: The type of the elements in the list. Must implement the
+              traits `EqualityComparable` and `CollectionElement`.
+
+        Args:
+            value: The value to find.
+
+        Returns:
+            True if the value is contained in the list, False otherwise.
+        """
+
+        constrained[
+            _type_is_eq[ElementType, C](), "value type is not self.ElementType"
+        ]()
+        for i in self:
+            if value == rebind[C](i[]):
```

Comment:
`rebind` doesn't work for mem types.

Reply:
@soraros, right, I missed this, can you put up a patch to fix this? Or to document this limitation?